neurogym/config/components/monitor.py

```diff
@@ -28,7 +27,8 @@ class MonitorLogConfig(ConfigBase):
     Attributes:
         verbose: Whether to enable verbose output.
         format: Log format string.
-        level: Logging level (e.g., INFO, DEBUG).
+        level: The minimum severity level from which logged messages should be sent to the sink.
```

Comment:
If we don't want to give the possibility to configure the logger (which is fine for me), then it makes sense to remove the `MonitorLogConfig` completely. At the moment only the `verbose` attribute is used in the entire code-base, and that could be moved to the `MonitorConfig` class.